registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -125,12 +125,9 @@ public class AbisHandlerStage extends MosipVerticleAPIManager {
 
 	@Value("${registration.processor.subscriber.id}")
 	private String subscriberId;
-
-	@Value("#{${mosip.regproc.abis.handler.biometric-modalities-segments-mapping}}")
-	private Map<String, List<String>> biometricModalitySegmentsMap;
 	
-	@Value("${mosip.regproc.abis.handler.status.code:REJECTED}")
-	private String statusCode;
+	@Value("${mosip.regproc.abis.handler.insufficient.biometrics.status.code:REJECTED}")

```

Comment:
"insufficient.biometrics.status.code" to be hyphen separated like "insufficient-biometrics-status-code"

Reply:
fixe